javascript/package.json

```diff
@@ -34,6 +34,8 @@
     "@semantic-release/github": "^8.0.5",
     "@semantic-release/npm": "^9.0.1",
     "@semantic-release/release-notes-generator": "^10.0.3",
+    "@stylistic/eslint-plugin": "^1.6.0",
```

Comment:
Would this need to be a real dependency instead?

Reply:
I'm not sure if it does in our case since we are re-exporting it as a package, but https://eslint.style/packages/js the install instructions didn't require it

Reply:
I think we would need it as a real dependency for the stylistic rules to work when we import this package? @chris-skoblenick can you help out with this?

Reply:
It's a `dependency` of this package.

For any users of this package, we require it to be downloaded.

A `devDependency` is something which is only needed for development *of this package*.

Reply:
updated to use dependency